### PR TITLE
Feature/incomingtransactions metrics

### PR DESCRIPTION
**Pull request by iariasleon**
```
                    SUMMARY:
 --------------------------------------------------------------------------------------------------------------------------------------------------
|                                           name                                               | passed | failed | skipped |  total  | duration(s) |
 --------------------------------------------------------------------------------------------------------------------------------------------------
| components\ngsiv2\admin\metrics\get_metrics\get_metrics_incoming_transactions_part_1.feature |     16 |      0 |       0 |      16 |      18.125 |
| components\ngsiv2\admin\metrics\get_metrics\get_metrics_incoming_transactions_part_2.feature |     12 |      0 |       0 |      12 |       9.032 |
 --------------------------------------------------------------------------------------------------------------------------------------------------
|  Date: Tuesday, January 24, 2017 11:20:22
 -------------------------------------------------
2 features passed, 0 failed, 0 skipped
28 scenarios passed, 0 failed, 0 skipped
706 steps passed, 0 failed, 0 skipped, 0 undefined
Took 0m27.157s

```

#### Coverage of requests
```
 GET     | /admin/log 
 PUT     | /admin/log  
 GET     | /admin/metrics       
 DELETE  | /admin/metrics                         
 GET     | /version  
 GET     | /statistics  
 GET     | /cache/statistics    
 GET     | /v2 
 GET     | /v2/entities/                       
 POST    | /v2/entities/                                      
 GET     | /v2/entities/`<entity_id>`   
 GET     | /v2/entities/`<entity_id>`/attrs
 POST    | /v2/entities/`<entity_id>`
 PATCH   | /v2/entities/`<entity_id>`
 PUT     | /v2/entities/`<entity_id>`
 DELETE  | /v2/entities/`<entity_id>`
 GET     | /v2/entities/`<entity_id>`/attrs/`<attr_name>`
 PUT     | /v2/entities/`<entity_id>`/attrs/`<attr_name>`
 DELETE  | /v2/entities/`<entity_id>`/attrs/`<attr_name>`
 GET     | /v2/entities/`<entity_id>`/attrs/`<attr_name>`/value
 PUT     | /v2/entities/`<entity_id>`/attrs/`<attr_name>`/value
 GET     | /v2/types/`<entity_type>`
 GET     | /v2/subscriptions
 POST    | /v2/subscriptions
 GET     | /v2/subscriptions/`<subscription_id>`
 PATCH   | /v2/subscriptions/`<subscription_id>`
 DELETE  | /v2/subscriptions/`<subscription_id>` 
 POST    | v2/op/update
 POST    | v2/op/query
 GET     | /log/trace                             | (pending)    
 PUT     | /log/trace/`<trace_levels>`            | (pending)   
 DELETE  | /log/trace/`<trace_levels>`            | (pending) 
 GET     | /admin/sem                             | (pending)    
 POST    | v2/op/register                         | (pending)  
 POST    | v2/op/discover                         | (pending)  
 GET     | /v2/registrations                      | (pending)  
 POST    | /v2/registrations                      | (pending)  
 GET     | /v2/registrations/`<registration_id>`  | (pending)  
 PATCH   | /v2/registrations/`<registration_id>`  | (pending)   
 DELETE  | /v2/registrations/`<registration_id>`  | (pending)  
```
@fgalan @ken

### CODE REVIEW

File: test/acceptance/behave/components/ngsiv2/admin/metrics/get_metrics/get_metrics_incoming_transactions_part_2.feature [hunk 1]
```diff
@@ -0,0 +1,659 @@
...
+#    incomingTransactionResponseSize: total size (bytes) in responses associated to incoming transactions (“out” from the point of view of the component).
+#    incomingTransactionErrors: number of incoming transactions resulting in error.
+#    serviceTime: average time to serve a transaction.
+#    outgoingTransactions: number of requests sent by the component.
+#    outgoingTransactionRequestSize: total size (bytes) in requests associated to outgoing transactions (“out” from the point of view of the component).
+#    outgoingTransactionResponseSize: total size (bytes) in responses associated to outgoing transactions (“in” from the point of view of the component).
+#    outgoingTransactionErrors: number of outgoing transactions resulting in error.
+#
+##############  WARNING:
+#   This tests suite must be executed complete, because the counters are accumulative in the same CB session. If you execute a scenario alone will fail.
```
Comment (kzangeli) [inline test/acceptance/behave/components/ngsiv2/admin/metrics/get_metrics/get_metrics_incoming_transactions_part_2.feature]: Perhaps you can remedy this by always resetting the counters? I.e. usse the `reset=true` option ... [ just brainstorming ]

Comment (iariasleon) [inline test/acceptance/behave/components/ngsiv2/admin/metrics/get_metrics/get_metrics_incoming_transactions_part_2.feature]: Yes, I think keeping the total counter is useful, however, below this line I explain how to use the reset parameter.

Comment (iariasleon) [inline test/acceptance/behave/components/ngsiv2/admin/metrics/get_metrics/get_metrics_incoming_transactions_part_2.feature]: I append new tests using the reset parameter

Comment (iariasleon) [inline test/acceptance/behave/components/ngsiv2/admin/metrics/get_metrics/get_metrics_incoming_transactions_part_2.feature]: fixed in  	129448a

**Comment by kzangeli** [comment]
LGTM

File: test/acceptance/behave/components/ngsiv2/admin/metrics/get_metrics/__init__.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+"""
+ Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
```
Comment (fgalan) [inline test/acceptance/behave/components/ngsiv2/admin/metrics/get_metrics/__init__.py]: 2016 -> 2017

(Only for new files, like this one)

Comment (iariasleon) [inline test/acceptance/behave/components/ngsiv2/admin/metrics/get_metrics/__init__.py]: fixed in dd68677 and  58506af

**Comment by fgalan** [comment]
LGTM

**Pull request merged**